tasks/cea-install.js

```diff
@@ -84,6 +85,28 @@ const srcInit = async () => {
   );
 };
 
+// Create the missing gitignore because npx wont copy it
```

Comment:
It's more that **npm** won't _publish_ it.  

Just clarify that and include the link to [`files` section in the CLI docs]() and this should be good enough to serve my memory 🙃 
https://docs.npmjs.com/files/package.json#files